app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -20,6 +20,19 @@ set -e
 <%= snippet 'puppet_setup' %>
 <% end -%>
 
+<% if host_param_true?('host_registration_insights') -%>
+echo '#'
+echo '# Installing Insights client'
+echo '#'
+
+yum install insights-client
+insights-client --register
+<% end -%>
+
+<% if host_param_true?('host_registration_rex') -%>
```

Comment:
As discussed today, we don't need to wrap this in parameter, kickstart also does not do it. The snippet already checks if the param is set. In some later PR, we'll just change the logic a bit to seed the param but default to false. But for this PR, let's not add it yet. (let's keep it for insights though, that is a new thing)